test-module/api/src/test/java/org/openmrs/module/testmodule/api/TestModuleServiceJUnit4Test.java

```diff
@@ -11,12 +11,15 @@
 import static org.hamcrest.Matchers.is;
 
 import org.junit.Test;
-import org.openmrs.test.BaseModuleContextSensitiveTest;
+import org.junit.runner.RunWith;
+import org.openmrs.test.jupiter.BaseModuleContextSensitiveTest;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 /**
  * Testing with JUnit 4
  */
+@RunWith(SpringJUnit4ClassRunner.class)
```

Comment:
You are adding a junit4 class runner on a class extending `org.openmrs.test.jupiter.BaseModuleContextSensitiveTest` which uses [junit5](https://github.com/openmrs/openmrs-core/blob/7160d91804f22a10166cd3cf1de50a9ba53030c2/api/src/test/java/org/openmrs/test/jupiter/BaseContextSensitiveTest.java#L130). Why?

Reply:
Thanks for thee catch, I was testing something